### ISSUE TITLE
[Ballerina Shell] Shell CLI integrator tests should be enabled

### ISSUE THREAD

**Issue by kdsuneraavinash** [ballerina-platform/ballerina-lang#28080]
**Description:**
Shell CLI tests were disabled by #28079  due to inconsistency with test case passing.
These tests need to be fixed and re-enabled.

**Steps to reproduce:**
Testing shell-cli in some computers will yield in following errors.
```
ballerina-shell-cli-test-suite > ballerina-shell-cli-test > io.ballerina.shell.cli.test.integration.FunctionEvaluatorTest > testEvaluatePointers STANDARD_ERROR
    Exception in thread "Thread-59" java.lang.AssertionError: Define function named add. expected [] but found [>
    ]
        at org.testng.Assert.fail(Assert.java:96)
        at org.testng.Assert.failNotEquals(Assert.java:776)
        at org.testng.Assert.assertEqualsImpl(Assert.java:137)
        at org.testng.Assert.assertEquals(Assert.java:118)
        at org.testng.Assert.assertEquals(Assert.java:453)
        at io.ballerina.shell.cli.test.base.TestIntegrator.run(TestIntegrator.java:102)

ballerina-shell-cli-test-suite > ballerina-shell-cli-test > io.ballerina.shell.cli.test.integration.FunctionEvaluatorTest > testEvaluatePointers FAILED
    java.io.IOError: java.io.IOException: Pipe broken
        at org.jline.keymap.BindingReader.readCharacter(BindingReader.java:143)
        at org.jline.keymap.BindingReader.readBinding(BindingReader.java:110)
        at org.jline.keymap.BindingReader.readBinding(BindingReader.java:61)
        at org.jline.reader.impl.LineReaderImpl.doReadBinding(LineReaderImpl.java:839)
        at org.jline.reader.impl.LineReaderImpl.readBinding(LineReaderImpl.java:859)
        at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:586)
        at org.jline.reader.impl.LineReaderImpl.readLine(LineReaderImpl.java:417)
        at io.ballerina.shell.cli.jline.DumbJlineTerminalAdapter.readLine(DumbJlineTerminalAdapter.java:44)
        at io.ballerina.shell.cli.BallerinaShell.run(BallerinaShell.java:101)
        at io.ballerina.shell.cli.ReplShellApplication.execute(ReplShellApplication.java:88)
        at io.ballerina.shell.cli.test.integration.AbstractIntegrationTest.test(AbstractIntegrationTest.java:63)
        at io.ballerina.shell.cli.test.integration.FunctionEvaluatorTest.testEvaluatePointers(FunctionEvaluatorTest.java:45)

        Caused by:
        java.io.IOException: Pipe broken
```

**Comment by DamithaSenevirathne** [comment]
@kdsuneraavinash can you please describe more about the statement "some computers". 
According to the  #31553 It seems with the current ballerina master, shell-cli tests are working in both Windows and Ubuntu instances.
Is passing tests for both Windows and Ubuntu instances sufficient to close this issue ?

**Comment by kdsuneraavinash** [comment]
The issue causing the error was fixed in #28144. We decided to enable tests cases later, after locally testing in mac/windows, but I think it later got forgotten. So it should be safe to enable test cases now I think.

**Comment by pubudu91** [comment]
Closing since tests were enabled by #31553
